src/plugins/reporter/utils.js

```diff
@@ -23,10 +23,10 @@ export const utils = {
   },
   cleanCode(str) {
     str = str
-        .replace(/\r\n?|[\n\u2028\u2029]/g, '\n')
-        .replace(/^\uFEFF/, '')
-        .replace(/^function *\(.*\) *{|\(.*\) *=> *{?/, '')
-        .replace(/\s+\}$/, '');
+      .replace(/\r\n?|[\n\u2028\u2029]/g, '\n')
```

Comment:
the IDE automatically fix the spaces but I'm not aware about the content. Looking at the [strip-bom code](https://github.com/sindresorhus/strip-bom/blob/master/index.js) it seems that it can replace only `.replace(/^\uFEFF/, '')` but no other replaces. Am I right?